modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
 +----------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
```

Comment:
I don't think we need to specify this is a procedure here, since this is not done for Deallocate databases or Reallocate databases even though both of those are procedures as well.